src/Common/Doctrine/Entity/Meta.php

```diff
@@ -170,11 +170,8 @@ public function update(
 
     public function __clone()
```

Comment:
can you add the :void typehint
did you also test it?

Reply:
- : void return type declaration is not allowed
- nope, didn't tested this

Reply:
can you maybe test this by writing a test for it :)